pkgs/tools/misc/aptly/default.nix

```diff
@@ -15,12 +15,17 @@ buildGoPackage rec {
   goPackagePath = "github.com/smira/aptly";
   goDeps = ./deps.nix;
 
+  buildInputs = [ makeWrapper ];
+
   postInstall = ''
     rm $bin/bin/man
+    wrapProgram "$bin/bin/aptly" \
+      --prefix PATH ":" "${gnupg1compat}/bin" \
```

Comment:
Usually we prefer squashing commits for simple changes but the maintainer can do the same when merging.